modules/virt-configuring-workload-update-methods.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/upgrading-virt.adoc
+
+[id="virt-configuring-workload-update-methods_{context}"]
+= Configuring workload update methods
+
+When you upgrade {VirtProductName}, VirtualMachineInstance (VMI) workloads, such as `virt-launcher` pods, update automatically by default. You can configure workload update strategies or opt out of workload updates by editing the `HyperConverged` custom resource (CR).
+
+.Prerequisites
+
+* To use live migration as an update method, you must first enable live migration in the cluster.
++
+[NOTE]
+====
+If a `VirtualMachineInstance` CR contains `evictionStrategy: LiveMigrate` and the VMI does not support live migration, the VMI will not update.
+====
+
+.Procedure
+
+. To open the `HyperConverged` CR in your default editor, run the following command:
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
+----
+
+. Edit the `workloadUpdateStrategy` stanza of the `HyperConverged` CR. For example:
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  workloadUpdateStrategy:
+    workloadUpdateMethods: <1>
+    - LiveMigrate <2>
```

Comment:
You might want to change workloadUpdateMethods to reflect the current default value for this field, given the other two fields in this example is showing default values.

Reply:
That crossed my mind as well, but I wanted to show all possible options in the example. I might leave it as-is for that reason, but I will take this into consideration!